### PR TITLE
post about UltraStar Play

### PR DESCRIPTION
**Pull request by achimmihca**
Hi folks,
this pull request adds a post about UltraStar Play as alternative to UltraStar Deluxe for mobile platforms (Android, iOS).

The game is already fun to play, although not yet as feature complete as UltraStar Deluxe.
On the other hand, UltraStar Play offers some unique features.

I would appreciate if it gets mentioned on the USDX website.

Thank you for creating UltraStar Deluxe!

Best wishes,
Andreas

### CODE REVIEW

File: _posts/2022-11-25-ultrastar-on-mobile-with-ultrastar-play.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+layout: post
+title:  "UltraStar Play: New Game Instance Supports Mobile Devices"
+date:   2022-11-25 22:12:00 +0200
+categories: news
+---
+Ever wanted to play UltraStar on your smartphone? Look no further!
+
+__UltraStar Play__ is a redevelopment of the UltraStar game using the Unity engine. This allows to build it not only for desktop platforms but also for Android and iOS.
```
Comment (basisbit) [inline _posts/2022-11-25-ultrastar-on-mobile-with-ultrastar-play.md]: `redevelopment` sounds a bit weird / denglish. Maybe something like "new developed game from scratch".

`supports smartphones, tablets, android TV, and similar platforms.`

File: _posts/2022-11-25-ultrastar-on-mobile-with-ultrastar-play.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+The game is already fun to play, although not yet as feature complete as UltraStar Deluxe.
```
Comment (basisbit) [inline _posts/2022-11-25-ultrastar-on-mobile-with-ultrastar-play.md]: maybe use however instead although

File: _posts/2022-11-25-ultrastar-on-mobile-with-ultrastar-play.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+On the other hand, UltraStar Play offers some unique features, for example easy smartphone connection and a rich song editor with drag'n'drop editing.
```
Comment (basisbit) [inline _posts/2022-11-25-ultrastar-on-mobile-with-ultrastar-play.md]: `On the other hand` is missing a "on one hand`. Maybe replace.

"comes with a companion app so you don't need to bring extra microphones but instead can use your phones"

**Comment by achimmihca** [comment]
I changed some phrases. Please check again.

**Review by basisbit — APPROVED** [review]
looks good to me. (will wait a few more days before merging it to give other people a chance to comment if they want)

**Pull request merged**